### PR TITLE
[READY] Cargo packet update and modification 

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs
Adds in 4 more crates
Medical Misc Kit, one of each medical kits - 1250
Medical sprays, two of each medical sprays - 2500
WT-550 Auto Rifle ammo crates non-lethal - 1500
WT-550 Auto Rifle ammo  Special -3000
Adds stimulant pills to medical gear crate
Removes lots of different things in crates
Adds in tons of different things in crates
[why]
Makes makes crates that were over priced more reachable , and adds in more items that should be in the crates.

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Proofs of testing
https://cdn.discordapp.com/attachments/499347891161137172/500873708617990145/unknown.png
https://cdn.discordapp.com/attachments/499347891161137172/500878618965180446/unknown.png
https://cdn.discordapp.com/attachments/499347891161137172/500878828676186112/unknown.png
https://cdn.discordapp.com/attachments/499347891161137172/500879104074186752/unknown.png

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -393,7 +396,10 @@
 	cost = 1500
 	contains = list(/obj/item/clothing/suit/armor/bulletproof,
 					/obj/item/clothing/suit/armor/bulletproof,
-					/obj/item/clothing/suit/armor/bulletproof)
+					/obj/item/clothing/suit/armor/bulletproof,
+					/obj/item/clothing/head/helmet/alt,
+					/obj/item/clothing/head/helmet/alt,
+					/obj/item/clothing/head/helmet/alt)
```
Comment (deathride58) [inline code/modules/cargo/packs.dm:408]: There's already a helmet crate

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -419,9 +425,10 @@
 
 /datum/supply_pack/security/armory/energy
 	name = "Energy Guns Crate"
-	desc = "Contains two Energy Guns, capable of firing both nonlethal and lethal blasts of light. Requires Armory access to open."
-	cost = 2500
+	desc = "Contains three Energy Guns, capable of firing both nonlethal and lethal blasts of light. Requires Armory access to open."
+	cost = 2750
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: That's a pretty minor price increase for a 1.5x increase in total firepower

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -465,7 +472,9 @@
 					/obj/item/tank/internals/plasma,
 					/obj/item/grenade/chem_grenade/incendiary,
 					/obj/item/grenade/chem_grenade/incendiary,
-					/obj/item/grenade/chem_grenade/incendiary)
+					/obj/item/grenade/chem_grenade/incendiary,
+					/obj/item/clothing/suit/fire/atmos,
+					/obj/item/clothing/head/hardhat/atmos)
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: Why tho. There's already a firesuit crate

File: code/modules/cargo/packs.dm [hunk 4]
```diff
@@ -534,13 +543,32 @@
...
+	contains = list(/obj/item/ammo_box/magazine/wt550m9/wtrubber,
+					/obj/item/ammo_box/magazine/wt550m9/wtrubber,
+					/obj/item/ammo_box/magazine/wt550m9/wtrubber,
+					/obj/item/ammo_box/magazine/wt550m9/wtrubber)
+	crate_name = "auto rifle ammo crate"
+
+/datum/supply_pack/security/armory/wt550ammo_special
+	name = "WT-550 Auto Rifle Special Ammo Crate"
+	desc = "Contains 2 20-round Armour Piercing and Incendiary magazines for the WT-550 Auto Rifle. Each magazine is designed to facilitate rapid tactical reloads. Requires Armory access to open."
+	cost = 3000
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: Pretty cheap for some pretty deadly rounds

File: code/modules/cargo/packs.dm [hunk 5]
```diff
@@ -1324,7 +1376,8 @@
 					/obj/item/reagent_containers/glass/rag,
 					/obj/item/grenade/chem_grenade/cleaner,
 					/obj/item/grenade/chem_grenade/cleaner,
-					/obj/item/grenade/chem_grenade/cleaner)
+					/obj/item/grenade/chem_grenade/cleaner,
+					/obj/item/soap/deluxe)
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: Why tho. The entire point of the deluxe soap is that it's captain-exclusive

File: code/modules/cargo/packs.dm [hunk 6]
```diff
@@ -1544,8 +1604,7 @@
 	contains = list(/obj/item/pizzabox/margherita,
 					/obj/item/pizzabox/mushroom,
 					/obj/item/pizzabox/meat,
-					/obj/item/pizzabox/vegetable,
-					/obj/item/pizzabox/pineapple)
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: 🗡

File: code/modules/cargo/packs.dm [hunk 7]
```diff
@@ -2135,7 +2225,6 @@
 					/obj/item/pen/fourcolor,
 					/obj/item/pen/fourcolor,
 					/obj/item/pen,
-					/obj/item/pen/fountain,
```
Comment (deathride58) [inline code/modules/cargo/packs.dm:2138]: Why tho

File: code/modules/cargo/packs.dm [hunk 8]
```diff
@@ -2144,14 +2233,16 @@
...
+					/obj/item/storage/briefcase)
 	crate_name = "bureaucracy crate"
 
 /datum/supply_pack/misc/fountainpens
 	name = "Calligraphy Crate"
 	desc = "Sign death warrants in style with these seven executive fountain pens."
 	cost = 700
-	contains = list(/obj/item/storage/box/fountainpens)
+	contains = list(/obj/item/storage/box/fountainpens,
+					/obj/item/paper_bin)
```
Comment (deathride58) [inline code/modules/cargo/packs.dm:2250]: why tho

**Review by deathride58 — CHANGES_REQUESTED** [review]
See above

File: code/modules/cargo/packs.dm (hunk 7 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm:2138]: Fountain pens are for the Calligraphy Crate

File: code/modules/cargo/packs.dm (hunk 8 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm:2250]: 500 for 7 pens is kinda useless

File: code/modules/cargo/packs.dm (hunk 6 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: <3

File: code/modules/cargo/packs.dm (hunk 5 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: There are many ways to get the soap, meta station has some in maints, you can get it form space, and some maps have them in bathrooms I think

File: code/modules/cargo/packs.dm (hunk 4 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: KK ill up the cost to 4500

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm:408]: https://cdn.discordapp.com/attachments/499347841756561436/501029245057892354/unknown.png
No there isnt.

File: code/modules/cargo/packs.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: Thats now how 1.5x works

File: code/modules/cargo/packs.dm (hunk 3 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: Thats for sec, so they can use it without dieing form fire...

File: code/modules/cargo/packs.dm [hunk 9]
```diff
@@ -129,9 +129,10 @@
 
 /datum/supply_pack/emergency/metalfoam
 	name = "Metal Foam Grenade Crate"
-	desc = "Seal up those pesky hull breaches with 7 Metal Foam Grenades."
+	desc = "Seal up those pesky hull breaches with 14 Metal Foam Grenades."
 	cost = 1000
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: Could at least double the cost of the crate to account for the doubled count of metal foam nades
```suggestion
	cost = 2000
```

File: code/modules/cargo/packs.dm [hunk 10]
```diff
@@ -393,7 +396,10 @@
 	cost = 1500
 	contains = list(/obj/item/clothing/suit/armor/bulletproof,
 					/obj/item/clothing/suit/armor/bulletproof,
-					/obj/item/clothing/suit/armor/bulletproof)
+					/obj/item/clothing/suit/armor/bulletproof,
+					/obj/item/clothing/head/helmet/alt,
+					/obj/item/clothing/head/helmet/alt,
```
Comment (deathride58) [inline code/modules/cargo/packs.dm:407]: Make a separate bulletproof helmet crate or double the cost

File: code/modules/cargo/packs.dm [hunk 11]
```diff
@@ -450,13 +457,13 @@
...
 	desc = "Contains four tracking implants. Requires Armory access to open."
-	cost = 2000
+	cost = 1000
 	contains = list(/obj/item/storage/box/trackimp)
 	crate_name = "tracking implant crate"
 
 /datum/supply_pack/security/armory/fire
 	name = "Incendiary Weapons Crate"
-	desc = "Burn, baby burn. Contains three incendiary grenades, three plasma canisters, and a flamethrower. Requires Armory access to open."
+	desc = "Burn, baby burn. Contains three incendiary grenades, three plasma canisters, an atmos fire suit and a flamethrower. Requires Armory access to open."
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: Description needs adjustment due to the removal of the atmos fire suit from the crate

File: code/modules/cargo/packs.dm [hunk 12]
```diff
@@ -1303,14 +1352,15 @@
...
 	cost = 2000
-	contains = list(/obj/item/stack/tile/noslip/thirty)
+	contains = list(/obj/item/stack/tile/noslip/thirty,
+					/obj/item/stack/tile/noslip/thirty)
 	crate_name = "high-traction floor tiles crate"
 
 /datum/supply_pack/service/janitor
 	name = "Janitorial Supplies Crate"
-	desc = "Fight back against dirt and grime with Nanotrasen's Janitorial Essentials(tm)! Contains three buckets, caution signs, and cleaner grenades. Also has a single mop, spray cleaner, rag, and trash bag."
+	desc = "Fight back against dirt and grime with Nanotrasen's Janitorial Essentials(tm)! Contains three buckets, caution signs, and cleaner grenades. Also has a single mop, spray cleaner, rag, deluxe soap and a trash bag."
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: Description needs adjustment

**Review by deathride58 — CHANGES_REQUESTED** [review]

File: code/modules/cargo/packs.dm [hunk 13]
```diff
@@ -396,6 +399,15 @@
 					/obj/item/clothing/suit/armor/bulletproof)
 	crate_name = "bulletproof armor crate"
 
+/datum/supply_pack/security/armory/bulletarmor
```
Comment (deathride58) [inline code/modules/cargo/packs.dm]: Duplicate typepath.
```suggestion
/datum/supply_pack/security/armory/bullethelmets
```

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Review by deathride58 — APPROVED** [review]

**Pull request merged**